src/platform/forms-system/test/config/helpers.js

```diff
@@ -42,3 +44,17 @@ export function axeCheck(component) {
     });
   });
 }
+
+const createStore = (data = {}) =>
+  createCommonStore({
+    form: () => ({ data }),
+  });
+export const uploadStore = createStore();
+
+export const uploadFeatureData = {
```

Comment:
The `uploadFeatureData` contains the `request_locked_pdf_password` feature flag. I called it `uploadFeatureData` because it was shorter than `requestLockedPdfPasswordFeatureData`. The `data` is in there because it is added to the `data` within the `DefinitionTester`.